Tests/FoundationEssentialsTests/FileManager/FileManagerTests.swift

```diff
@@ -1072,4 +1072,90 @@ final class FileManagerTests : XCTestCase {
             XCTAssertEqual($0.contents(atPath: "a\u{301}/test"), data)
         }
     }
+
+    /// Tests that Foundation can correctly handle "long paths" (paths of 260 to 32767 chacters long) on Windows.
+    func testWindowsLongPathSupport() throws {
+        #if !os(Windows)
+        throw XCTSkip("This test is not applicable for this platform")
+        #endif
```

Comment:
Could we change this to `#else` and put the rest of this function in the else block? The current spelling produces this warning on non-Windows platforms which would be nice to avoid:

```
/build/swift-foundation/Tests/FoundationEssentialsTests/FileManager/FileManagerTests.swift:1084:9: warning: code after 'throw' will never be executed
```

Reply:
Done!